docs_src/index.md

```diff
@@ -168,7 +168,7 @@ SDKs are language specific; meaning an SDK is written to create services in a pa
 
 - Golang Device Service SDK
 - C Device Service SDK
-- Golang Device Service SDK
+- Golang Application Service SDK
```

Comment:
we call it `Golang Application Functions SDK`
https://docs.edgexfoundry.org/1.3/microservices/application/ApplicationFunctionsSDK/

Reply:
Ok.